Src/zipkin4net/Src/EndPoint.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Net;
+
+namespace zipkin4net
+{
+    //TODO internal for now. It's still work in progress
+    internal class EndPoint : IEndPoint
```

Comment:
nit endpoint not endPoint (just to make sure the case format doesn't bleed through to marshalling)